integrations/operator/main.go

```diff
@@ -41,7 +41,6 @@ import (
 	resourcesv5 "github.com/gravitational/teleport/integrations/operator/apis/resources/v5"
 	"github.com/gravitational/teleport/integrations/operator/controllers/resources"
 	sidecar2 "github.com/gravitational/teleport/integrations/operator/sidecar"
-	//+kubebuilder:scaffold:imports
```

Comment:
same question, is this related to the PR?

Reply:
Same!